vm_trace.c

```diff
@@ -1195,6 +1195,10 @@ rb_tracepoint_enable(VALUE tpval)
         rb_raise(rb_eArgError, "can't nest-enable a targeting TracePoint");
     }
 
+    if (tp->tracing) {
+        return Qundef;
```

Comment:
I think this should return `tpval` and not `Qundef`.

Reply:
Originally, this method returned `Qundef`, so I chose the same return value for the newly added branch.
I think it would be possible to modify it to return `tpval`, or to return `Qtrue` or `Qfalse` depending on whether a new hook is added or not.
However, it would be necessary to confirm if such a change is acceptable.

Reply:
Wow, you're totally right I didn't catch that.  I'll just merge this, thank you!